huntsman/camera/pyro.py

```diff
@@ -129,6 +129,9 @@ def is_exposing(self):
     def is_ready(self):
         '''
         True if camera is ready to start another exposure, otherwise False.
+
+        Check self.is_exposing first to side-step so-far unexplained
+        hanging possibly caused by is_temperature_stable.
```

Comment:
I don't think this comment should be in the docstring for this property. It strongly suggests to the user that they are responsible for checking `is_exposing` first before checking `is_ready`, but my previous PR built that check in to the `is_ready()` method (immediately below this docstring!) so that would now be redundant.